kms/keysource.go

```diff
@@ -165,8 +165,13 @@ func (key MasterKey) createStsSession(config aws.Config, sess *session.Session)
 	if err != nil {
 		return nil, err
 	}
+	stsRoleSessionNameRe, err := regexp.Compile("[^a-zA-Z0-9=,.@-]+")
+	if err != nil {
+		return nil, err
```

Comment:
Please return a more meaningful error here. Wrap it and add some context so users know what it’s about.

Reply:
@autrilla please take another look